src/GPLikelihoods.jl

```diff
@@ -4,7 +4,7 @@ using Distributions
 using AbstractGPs
 using Random
 using Functors
-using StatsFuns: logistic, softmax
+using StatsFuns: logistic
```

Comment:
We could think about using LogExpFunctions instead: https://github.com/JuliaStats/StatsFuns.jl/pull/108